Keas.Mvc/ClientApp/components/Equipment/EquipmentTable.tsx

```diff
@@ -32,7 +32,7 @@ export default class EquipmentTable extends React.Component<IProps, {}> {
             Details
           </Button>
         ),
-        Header: 'Equipment Actions',
+        Header: 'Details',
```

Comment:
Same as above. If possible, I think this would look better without a header.

Reply:
Same as previous comment, without the a header, the application will break